vendor-extra/JatsContentBundle/src/EventListener/BodyListener.php

```diff
@@ -21,9 +21,13 @@ public function __construct(ViewConverter $converter)
         $this->converter = $converter;
     }
 
-    public function onCreatePage(CreateContentPageEvent $event) : void
+    public function onCreatePage(CreatePageEvent $event) : void
     {
-        $xpath = $event->getItem()->xpath();
+        if ('content' !== $event->getRequest()->attributes->get('libero_page')['type']) {
```

Comment:
It'd be useful to make this so listeners aren't called for things they don't care about.